spec/system/task_spec.rb

```diff
@@ -1,24 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { FactoryBot.create(:project) }
+  let(:task) { FactoryBot.create(:task) }
```

Comment:
`FactoryBot.`の定義は設定ファイルに追記すると省略できますね。
```suggestion
  let(:task) { create(:task) }
```

Reply:
4525039 FactoryBotを省略しました。